immuneML/reports/encoding_reports/FeatureComparison.py

```diff
@@ -0,0 +1,156 @@
+import warnings
+from pathlib import Path
+
+import plotly.express as px
+import pandas as pd
+
+from immuneML.data_model.dataset.Dataset import Dataset
+from immuneML.reports.ReportOutput import ReportOutput
+from immuneML.reports.encoding_reports.FeatureReport import FeatureReport
+
+
+class FeatureComparison(FeatureReport):
+    """
+    Compares the feature values in a given encoded data matrix across two values for a metadata label.
```

Comment:
maybe it would be nice to define what is a 'metadata label' - where it can be found etc?

Reply:
it is just the same as 'label', which is a term we use everywhere. Would it be clearer if the word 'metadata' is omitted?

Reply:
hm i am not sure.. it was a bit confusing initially but makes sense. maybe half a sentence that it is the same like the fields in the metadata file? but I am not sure, so whatever you think is best :)